package.json

```diff
@@ -96,7 +96,7 @@
     "@types/prop-types": "^15.7.3",
     "@types/react": "^16.14.2",
     "@types/react-dom": "^16.9.10",
-    "@types/sql-formatter": "^2.3.0",
+    "@types/sql-formatter": "^4.0.1",
```

Comment:
From https://www.npmjs.com/package/@types/sql-formatter, it seems sql-formatter now provides its own types? Probably can remove this package, which may be what's failing CI

Reply:
@gabrieldutra 
Thanks for the review.
You are correct, removing @types/sql-formatter did not affect the behavior.